src/core/tsi/alts/crypt/aes_gcm.cc

```diff
@@ -655,12 +665,10 @@ static grpc_status_code aes_gcm_new_evp_cipher_ctx(
   return GRPC_STATUS_OK;
 }
 
-grpc_status_code gsec_aes_gcm_aead_crypter_create(const uint8_t* key,
-                                                  size_t key_length,
-                                                  size_t nonce_length,
-                                                  size_t tag_length, bool rekey,
-                                                  gsec_aead_crypter** crypter,
-                                                  char** error_details) {
+grpc_status_code gsec_aes_gcm_aead_crypter_create(
```

Comment:
I think we missed a copy of the key in the `aes_gcm_new_evp_cipher_ctx` function. Is there an easy way to deal with that buffer? I'm also ok doing this in a follow-up if you'd like.

Reply:
This is again a local buffer that is similar to the rekey we discussed earlier. I added the same TODO here.